portal-api/src/v1/controllers/feedback.controller.js

```diff
@@ -38,18 +36,15 @@ exports.create = async (req, res) => {
 
   const modifiedFeedback = {
     ...req.body,
-    submittedBy: req.user.username,
-    created: now(),
+    created: moment().unix(),
```

Comment:
can we use date-fns instead or Date.now()?

Reply:
done

Reply:
I can't see the changes here

Reply:
I forgot to push :P